### DISCUSSION TITLE
iOS add Controller Center support

### DISCUSSION THREAD

**Discussion by cbjn** [mullvad/mullvadvpn-app#9061] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

iOS 18 Contriller widget vpn connection switch support 

### Alternative solutions

more Feature

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [X] iOS
- [ ] Windows
- [ ] macOS
- [ ] Linux

**Comment by artemvmin** [comment]
The title is confusing, but I assume this refers to the Control Center. I would love to get a simple toggle button. iOS already does for their system VPN setting, as do some other VPN providers. For a great example, check out Tailscale.

This also should solve the staleness / limited update budget issue described in https://github.com/mullvad/mullvadvpn-app/issues/3096#issuecomment-1689986601
